databags/community_team_members.json

```diff
@@ -1,5 +1,55 @@
 {
     "projects": [
+        {
+            "members": [
+                {
+                    "name": "Arushi Verma",
+                    "role": "Community Contributor"
+                },
+                {
+                    "name": "Saurabh Argarwala",
+                    "role": "Community Collaborator"
+                }
+            ],
+            "name": "Community Roles",
```

Comment:
I don't think it makes sense for this to be another project. I think we should keep the "projects" top level key for project-based roles and have a separate top-level key for community roles. we should also render community roles differently on the page since they're not related to projects.

See my comment here for how I think they should be rendered (open to discussion): https://github.com/creativecommons/creativecommons.github.io-source/pull/244#issuecomment-626947946